sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsublite/SubscriberOptions.java

```diff
@@ -86,11 +107,31 @@
 
   public static Builder newBuilder() {
     Builder builder = new AutoValue_SubscriberOptions.Builder();
-    return builder.setPartitions(ImmutableSet.of()).setFlowControlSettings(DEFAULT_FLOW_CONTROL);
+    return builder
+        .setPartitions(ImmutableSet.of())
+        .setFlowControlSettings(DEFAULT_FLOW_CONTROL)
+        .setMinBundleTimeout(MIN_BUNDLE_TIMEOUT);
   }
 
   public abstract Builder toBuilder();
 
+  private SubscriberServiceClient newSubscriberServiceClient(Partition partition)
+      throws ApiException {
+    try {
+      SubscriberServiceSettings.Builder settingsBuilder = SubscriberServiceSettings.newBuilder();
+
+      settingsBuilder =
+          addDefaultMetadata(
+              PubsubContext.of(FRAMEWORK),
+              RoutingMetadata.of(subscriptionPath(), partition),
+              settingsBuilder);
```

Comment:
Memo: This assignment to `settingsBuilder` is unnecessary, but without that error prone gives error for ignoring the return value of the method.